network-api/networkapi/settings.py

```diff
@@ -411,7 +411,7 @@
 LANGUAGES = (
     ('en', gettext_lazy('English')),
     ('de', gettext_lazy('German')),
-    ('pt', gettext_lazy('Portuguese')),
+    ('pt-BR', gettext_lazy('Portuguese')),
```

Comment:
@TheoChevalier should this be `Brazilian Portuguese`?

Reply:
Yeah we should flip this switch, too. I think we should go with `Portuguese (Brazil)`, like mozilla.org, just so that it doesn’t mess around with the alpha sorting

Reply:
```suggestion
    ('pt-BR', gettext_lazy('Portuguese (Brazil)')),
```